enzy_htp/structure/structure_operation/general.py

```diff
@@ -49,6 +49,9 @@ def remove_non_peptide(stru: Structure) -> Structure:
         ch.delete_from_parent()
     return stru
 
+def remove_hydrogens(ligand: Ligand, polypeptide_only: bool) -> None:
```

Comment:
Please make it general to Residues and dont put these fix in the unit test PR

Reply:
The modifications to files in `enzyhtp` directory are now unstaged.